glosses.html

```diff
@@ -54,6 +54,39 @@
             setTimeout(() => location.href = "", 3000)
         })
 
+        function removeDuplicateDeerIds() {
```

Comment:
You do not need any of the code on this page.  As am I testing locally, I deleted all of this and it functions just fine.  Delete this from the PR.